app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t("static_pages.home.title"))%>
+  <h1><%= t "static_pages.home.content_home" %></h1>
+  <p>
+    <%= t "static_pages.home.content_home1"  %>
+    <%= link_to t("static_pages.home.content_home2"), "http://www.railstutorial.org/" %>
+      <%= t "static_pages.home.content_home2"  %>
```

Comment:
[IMO] This line is redundant because we already have link above. And we need to remove `</a>` below